src/DataStructures/Tensor/Expressions/TensorIndex.hpp

```diff
@@ -146,13 +146,14 @@ get_tensorindex_value_with_opposite_valence(const size_t i) {
  * integer ranges to properly encode the new index (both upper and lower
  * definitions) that you wish to add. In short, you should simply be able to
  * continue the pattern used for the existing `TensorIndex` types that are
- * already defined. For example, if `ti_M`/`ti_m` is the highest-valued generic
```

Comment:
why reword this? I feel like the documentation isn't great if needs to be reworded each time a new index is added... Is it really confusing to leave as-is?

Reply:
I think you're right that we can leave it. I'll undo the rewording here!